src/pybind/mgr/prometheus/module.py

```diff
@@ -1811,6 +1861,18 @@ def _clear_healthchecks(self) -> HandleCommandResult:
         self.health_history.reset()
         return HandleCommandResult(retval=0, stdout="healthcheck history cleared")
 
+    @CLIWriteCommand('prometheus dump cacert')
```

Comment:
If this is just a command for cephadm use, I'd replace it with a `config-key get`. If not, I'd put a more user-friendly name: `prometheus certs dump` or something (90% of the time Ceph CLI keeps the noun-action syntax, instead of the action-noun).

Reply:
Removed as now we use config-key to get the certificate.